httpservice/include/cppmicroservices/httpservice/HttpServletRequest.h

```diff
@@ -38,15 +38,14 @@ struct HttpServletRequestPrivate;
 class US_HttpService_EXPORT HttpServletRequest
 {
 public:
-
   ~HttpServletRequest();
 
-  HttpServletRequest(const HttpServletRequest& o);
-  HttpServletRequest& operator=(const HttpServletRequest& o);
+  HttpServletRequest(const HttpServletRequest &o);
```

Comment:
Hi, first, thanks for this PR!

The first thing that jumps out is the various changes that puts the pointer and reference identifiers (* and &) next to the variables. We don't explicitly mention - but we prefer letting these identifiers stay adjacent to the types. If you look at the framework code, the convention we follow is these identifiers should be next to the type.

Reply:
Hi, 
Is it possible to provide needed code convention? we are using clang-format to automatically format the code and hence we can provide the config file

Reply:
https://zed0.co.uk/clang-format-configurator/ can be used to quickly create/check a format file

Reply:
Thanks, this is definitely in the cards for us - to create a project-wide clang-format.